tasks/oah-install.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Checking the latest version
+  shell: 'curl -s https://api.github.com/repos/opensbom-generator/spdx-sbom-generator/releases/latest | grep "browser_download_url.*linux-amd64.tar.gz" | cut -d : -f 2,3 | tr -d \" | head -1 | xargs'
+  register: latest_version
+  warn: false
+
+- name: Download and Extract spdx-generator tool

```

Comment:
Require sudo

Reply:
Added the required statement

Reply:
Added the required sudo statement for curs -s also